includes/polling/os/fortiwlc.inc.php

```diff
@@ -0,0 +1,4 @@
+<?php
+
+$version = snmp_get($device, "MERU-GLOBAL-STATISTICS-MIB::mwSystemGeneralVersion.0", "-OQv");
```

Comment:
Can you convert these two `snmp_get()` calls into one `snmp_get_multi_oid()`. You also don't and shouldn't prepend the MIB name to the OID, pass it as an arg to the function.